src/main/java/uk/go/hm/icb/dto/ICBMatch.java

```diff
@@ -23,7 +23,8 @@ public ICBMatchBuilder matches(String firstNameMatched, String lastNameMatched,
                     Pair.of("Address", addressMatched),
                     Pair.of("Unique Identifier - Birth Cert", birthCertificateMatched),
                     Pair.of("Driving Licence Number", drivingLicenseNumberMatched),
-                    Pair.of("Passport Number", passportNumber)
+                    Pair.of("Passport Number", passportNumber),
```

Comment:
Do you want to add logic for adding flag? i.e: only if flag value is there.. otherwise don't add  it to the list?